src/api/event-services/work-request.js

```diff
@@ -43,6 +57,7 @@ const handleWorkRequest = async (workRequest, socket) => {
     } else {
       logger.log('Unexpected error happened while trying to process cached response:', e.message);
```

Comment:
Hmm.. not really. At this stage, there might be expected errors (from the pipeline) and unexpected errors (from parsing the work response). This can be changed into something more neutral.